domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/VurderAvslagGrunnetBeregning.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.su.se.bakover.domain.behandling
+
+import no.nav.su.se.bakover.domain.behandling.avslag.Avslagsgrunn
+import no.nav.su.se.bakover.domain.beregning.Beregning
+import no.nav.su.se.bakover.domain.beregning.Sats
+
+object VurderAvslagGrunnetBeregning {
+    fun vurderAvslagGrunnetBeregning(
+        beregning: Beregning?
+    ): AvslagGrunnetBeregning = if (beregning == null) AvslagGrunnetBeregning.Nei else when {
+        beregning.getSumYtelse() <= 0 -> AvslagGrunnetBeregning.Ja(Avslagsgrunn.FOR_HØY_INNTEKT)
+        harMånederUnderMinstebeløp(beregning) -> AvslagGrunnetBeregning.Ja(Avslagsgrunn.SU_UNDER_MINSTEGRENSE)
```

Comment:
Dette leser veldig bra :+1: 

Jeg har ikke fulgt med i diskusjonen rundt avslag dersom man tjener for mye en av månedene ila. perioden. Men det er sånn at vi skal avslå hele perioden dersom dette er tilfelle?

Reply:
Det er linja vi har fått beskjed om å legge oss på inntil videre.